docs/getting-started/concepts.md

```diff
@@ -7,67 +7,67 @@ Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data
 
 ## Use-cases
 
-Iter8 simplifies a variety of traffic engineering and metrics-driven validation use-cases. They are illustrated below.
+Iter8 simplifies a variety of traffic engineering and metrics-driven validation use-cases. To support such use cases, Iter8 provides support for the key challenges enabling simpler implementation and quicker adoption.
```

Comment:
```suggestion
Iter8 simplifies a variety of traffic engineering and metrics-driven validation use cases. To support such use cases, Iter8 provides support for the key challenges enabling simpler implementation and quicker adoption.
```

In these two sentence, you've used both `use-cases` and `use cases`. I think the latter is fine

Reply:
> To support such use cases, Iter8 provides support for the key challenges enabling simpler implementation and quicker adoption.

I think this last sentence is a bit vague. What key challenges? What simpler implementation" Quicker adoption of what?

Reply:
updated